Makefile

```diff
@@ -101,6 +101,7 @@ e2e-test: kuttl-shfmt
 manifests: kustomize generate
 	$(KUSTOMIZE) build config/crd/ > $(DEPLOYDIR)/crd.yaml
 	echo "---" >> $(DEPLOYDIR)/crd.yaml
+
```

Comment:
is this necessary?

Reply:
Just wanted to split CRDs from rbac and operator generation. Nothing special